cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -124,7 +124,7 @@ fn count_zero_half_words(mut value: u64, num_half_words: u8) -> usize {
 fn inst_size_test() {
     // This test will help with unintentionally growing the size
     // of the Inst enum.
-    assert_eq!(32, std::mem::size_of::<Inst>());
+    assert_eq!(40, std::mem::size_of::<Inst>());
```

Comment:
We probably don't want to increase the size of `Inst` if we can help it at all -- the point of this test is to guard against perf regressions. Can we find a way to remove a field or, failing that, box the contents of the inst? Two options come to mind:

- (Maybe preferred) eliminate `rtmp`, and use `x1` explicitly instead. This is part of the clobber-set of the call already, so it should be legal to use it to hold the address of the called function.
- (Alternately) create a struct and hold a `Box` in the inst, taking only 8 bytes

Reply:
Ah, I wasn't sure that it was okay to use `x1`, but your explanation makes sense (and helped clear up my confusion) - thanks!

Just pushed a commit to make that change (and reverted the change to the size test as well)